lib/rollbax/item.ex

```diff
@@ -27,7 +27,7 @@ defmodule Rollbax.Item do
     if map_size(meta) == 0 do
       data
     else
-      Map.put(data, "custom", meta)
+      Map.merge(data, meta)
```

Comment:
You're talking about something like `Rollbax.report(..., %{my_thing: 123, root_data: %{person: %{id: 123}}})` which would result in `%{env: "as_usual", person: %{id: 123}, custom: %{my_thing: 123}}` to be reported to rollbar? if so, i still think it might be a better solution to just keep it simple and let users overwrite the defaults.

also, i don't see the problem with `uuid`. i would assume users wants to have the uuid field in the payload if they explicitly provide it...

anyway, as long as i'll be able to set person and request fields i'll be happy. and having Logger.error reports sent to rollbar is just really great. that was already incredibly useful for me. thanks for that!